src/csharp/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Spark.E2ETest;
+using Microsoft.Spark.E2ETest.Utils;
+using Microsoft.Spark.Extensions.Delta.Tables;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.Extensions.Delta.E2ETest
+{
+    /// <summary>
+    /// These tests will fail if the Delta Jar is not installed in the 
+    /// $SPARK_HOME/jars/ directory.
```

Comment:
So, extra step will be required? Why not update SparkFixture such that it automatically download the jar using `--packages`?

Reply:
Changed. `SparkFixture` now provides `--packages` param.